### PR TITLE
Improvements for Mermaid support

### PR DESCRIPTION
**Pull request by sftim**
- make Mermaid message about JavaScript localizable
- improve message rendering
- upgrade to Mermaid 8.13.0

/area web-development

### CODE REVIEW

**Comment by sftim** [comment]
Tested locally, with and without JavaScript.

**Comment by sftim** [comment]
This seems to help with #31055 too.

File: layouts/partials/scripts.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
 <script src="/js/bootstrap-4.3.1.min.js" integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM" crossorigin="anonymous"></script>
 
 {{ if .Site.Params.mermaid.enable }}
-<script src="/js//mermaid.min.js" crossorigin="anonymous"></script>
+<script src="/js/mermaid.min.js" crossorigin="anonymous"></script>
```
Comment (championshuttler) [inline layouts/partials/scripts.html:6]: 💯 this should help in rendering graphs directly in our codebase

**Review by championshuttler — COMMENTED** [review]
thanks for working on it

**Comment by kbhawkey** [comment]
Thanks @sftim .
/lgtm

**Pull request merged**